poc/gen_test_vec.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+# /usr/bin/env python3
```

Comment:
This change was suggested by autopep. In fact, this file's permissions don't include exec, so we can probably just delete this line?

Reply:
oh, it knows more about formatting than I do :) either removing or just this change is fine.

Reply:
Removed.